clang/test/Modules/compare-file-size.py

```diff
@@ -0,0 +1,22 @@
+import argparse
```

Comment:
does this require a license/copyright header?  

I see a `#!/usr/bin/env python` in https://github.com/llvm/llvm-project/blob/cdd11d694a406a98a16d6265168ee2fbe1b6a87c/llvm/utils/lit/tests/Inputs/shtest-env/print_environment.py#L1, do yo need something similar here? You're using python to invoke, so it's probably fine, but it seems a bit inconsistent. Do you know what's more typical in the test support scripts like this?

Reply:
@pogo59 You mentioned in the [RFC](https://discourse.llvm.org/t/rfc-enabling-the-lit-internal-shell-by-default/80179/29) that when there isn't another good option that a python script may be an OK substitute in some cases. I'm only aware of a couple of those in the tree(mostly in lit's own test directory). Do you know if these require a license header, or if we have a good convention for wether to use `#!/bin/env python` or not.

Reply:
There is a lit substitution `%python` and I see a fair number of these in the lit tests. Some use inline python scripts, some have separate scripts, and a number of them have the python script directly in the test file (`RUN: %python %s ...`). It's far more common in llvm than clang, so you may have better luck looking there for inspiration.

Test files generally do not have license headers.

Reply:
> Test files generally do not have license headers. 

Ah, right. I was thinking of them differently, since they're not par to of the test per se, but now that you spell it out, that makes sense. Thanks.